connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfigTransformer.java

```diff
@@ -98,4 +101,8 @@ public void onCompletion(Throwable error, Void result) {
         HerderRequest request = worker.herder().restartConnector(ttl, connectorName, cb);
         connectorRequests.put(path, request);
     }
+
+    public void close() {
```

Comment:
should we also change this class to implement `AutoCloseable`? 
This can't be used immediately in a try-with-resources clause, but probably better to signal the existence of this method at the class level.